.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
+labels: bug
+---
+
+**Commit version**
+
+The commit version goes here.
```

Comment:
I am not sure the users can easily tell which commit version they use, should we remove this part?